src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Create.cs

```diff
@@ -90,7 +90,7 @@ private static SyntaxTreeIndex CreateIndex(
 
                         containsForEachStatement = containsForEachStatement || syntaxFacts.IsForEachStatement(node);
                         containsLockStatement = containsLockStatement || syntaxFacts.IsLockStatement(node);
-                        containsUsingStatement = containsUsingStatement || syntaxFacts.IsUsingStatement(node);
+                        containsUsingStatement = containsUsingStatement || syntaxFacts.IsUsingStatement(node) || syntaxFacts.IsUsingLocalDeclarationStatement(node);
```

Comment:
>IsUsingLocalDeclarationStatement [](http://example.com/codeflow?start=125&length=32)

To clarify for me, this is tracking both "using namespace" and "using var foo = IDisposable" statements, right? Other than the name being the same, why are they grouped together?

Reply:
> To clarify for me, this is tracking both "using namespace" and "using var foo = IDisposable" statements, right? 

Yes.

> Other than the name being the same, why are they grouped together?

Because this is for FindRefs, which only cares about tthis info just to say "should i look for Dispose references here?".  And for FindRefs, it doesn't care about which of the two cases it is.

I could do this as two different bools, but the consumption side would then have to check for both.  So i just collapse here to one bool since htat's all we need.